src/components/ResultsBody/PaginatedList/VenueCard/venueCard.module.css

```diff
@@ -0,0 +1,75 @@
+.wrapper {
+  min-width: 282px;
+  min-height: 228px;
+  box-shadow: 0 0 0.4vw var(--secondary-light);
+}
+
+:root {
+  --background-url: url("default-image.jpg");
+}
+
+.pictureFrame {
+  height: 80%;
+  width: 100%;
+  background-size: cover;
+  background-position: center;
+  position: relative;
+  background-image: var(--background-url);
+}
+
+.infoBar {
+  background-color: var(--background-gray);
+  min-height: 28px;
+  width: 100%;
+  height: 20%;
+}
+
+.heartBox {
+  width: 2.22vw;
+  height: 2.22vw;
+  background-color: black;
+  opacity: 0.63;
+  position: absolute;
+  top: 0;
+  border-radius: 0 0 10px 0;
+}
+
+.nameBox {
+  height: 2.22vw;
+  width: fit-content;
+  background-color: black;
+  opacity: 0.63;
+  position: absolute;
+  top: 0;
+  right: 0;
+  border-radius: 0 0 0 20px;
+  padding: 0.3vw 1.11vw 0.3vw 2.02vw;
+  display: inline-block;
+  text-transform: capitalize;
+}
+
+.infoBox {
+  height: 4.86vw;
+  width: 9.72vw;
+  background-color: black;
+  opacity: 0.63;
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  border-radius: 0 20px 0 0;
+  padding: 0.3vw 1vw 0.3vw 1vw;
```

Comment:
Padding should not be dependant on page size fluently. 

Basicaly don't use vw. 
vw is good for setting body width. In other cases it should be used rarely.

Reply:
done